core/templates/pages/release-coordinator-page/beam-jobs-tab/beam-jobs-tab.component.ts

```diff
@@ -46,14 +46,17 @@ export class BeamJobsTabComponent implements OnInit, OnDestroy {
   jobNameControl = new FormControl('');
 
   dataIsReady = false;
-  beamJobs: BeamJob[] = null;
-  selectedJob: BeamJob = null;
-
+  // This property is initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion. For more information, see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  beamJobs!: BeamJob[];
+  filteredJobNames!: Observable<string[]>;
+  filteredBeamJobRuns!: Observable<BeamJobRun[]>;
+  beamJobRunsRefreshIntervalSubscription!: Subscription;
+  // Selected job is null if no job is selected.
+  selectedJob: BeamJob | null | undefined = null;
```

Comment:
Why do we allow both `null` and `undefined`.

Reply:
Allow `undefined` only now. Not using null because while selecting targeting job (beamJob we need) from all beamJobs, if we unable to find, it throws undefined, which is further use to make button disable to start job.